### ISSUE TITLE
Passing Store to RecipeList while Testing.

### ISSUE THREAD

**Issue by asimashfaq** [pylnata/recipies-app#1]
Hi, I would like to know about how you able to pass the store object to the RecipeList component. I had to wrap up my component inside the Provider.

Thanks

**Comment by pylnata** [comment]
Are you asking about test? Here is explanation https://medium.com/@pylnata/testing-react-functional-component-using-hooks-useeffect-usedispatch-and-useselector-in-shallow-9cfbc74f62fb

**Comment by asimashfaq** [comment]
Yes, you can, or even in your code, you're using the same things with little bit difference. So I decided to ask you and take your help. As your work Impress me.

In my project, this line is not working for My Component

` wrapper = shallow(<RecipeList store={store} />); `

I got this error. ` Type '{ store: any; }' has no properties in common with type 'IntrinsicAttributes & { children?: ReactNode; }'.`

**Comment by pylnata** [comment]
It means that you have another type for component. In my case RecipeList has type React.FC. You should adjust yours

**Comment by asimashfaq** [comment]
Yes i did that but its not working.

**Comment by vishwa3** [comment]
@pylnata : I have a query but not related to the above issue and I don't think my query is an issue perse so asking on this thread . Why didn't you add tests for event handlers like onPageClickHandler in your RecipeList.tsx  . Is it because methods cant be tested for functional components using JEST/Enzyme

**Comment by pylnata** [comment]
Hi, @vishwa3 I didn't have goal to cover all functionality by tests. If to talk about onPageClickHandler it is definitely  possible to write test using JEST/Enzyme -  test to check that it dispatches correct action with correct params on redux store . I encourage you to do it by yourself :)
